components/tooltip/Tooltip.razor.cs

```diff
@@ -11,6 +11,9 @@ public partial class Tooltip : OverlayTrigger
         [Parameter]
         public OneOf<string, RenderFragment> Title { get; set; } = string.Empty;
 
+        [Parameter]
+        public bool IsMarkupString { get; set; } = false;
+
```

Comment:
add `MarkupString` in `Title` type, just like `OneOf<string, MarkupString ,RenderFragment>`

Reply:
modified